packages/ui/components/scheduler/components/blocking/BlockedTimeCell.tsx

```diff
@@ -0,0 +1,15 @@
+import { classNames } from "@calcom/lib";
+
+export function BlockedTimeCell() {
+  return (
+    <div
+      className={classNames("group absolute inset-0 flex h-full flex-col  hover:cursor-not-allowed")}
```

Comment:
`hover:cursor-not-allowed` to give some more indication that the user is not meant to be clicking on these spots.